api/src/pcapi/routes/serialization/base.py

```diff
@@ -134,3 +136,13 @@ class ListOffersVenueResponseModel(BaseModel):
     offererName: str
     publicName: str | None
     departementCode: str | None
+
+
+class VenueOpeningHoursModel(BaseModel):
+    weekday: str
+    timespan: list[list[str]] | None
+
+    @validator("timespan", each_item=True)
+    def convert_to_numeric_ranges(cls, timespan: list[str]) -> NumericRange:
+        start, end = timespan
+        return NumericRange(time_to_int(start), time_to_int(end), "[]")
```

Comment:
Je suis désolé, je ne comprends pas bien : pourquoi `timespan` n'est pas une liste de `NumericRange` dans ce cas ? Voire même une liste d'éléments avec deux champs chacun : `start` et `end` ?

Reply:
C'est l'idée du `each_item=True`: cela permet d'itérer sur l'ensemble des champs de timespan. On renvoie donc bien une liste de NumericRange(lower=start, upper=end, bounds="[]")